Plugin.php

```diff
@@ -69,14 +69,23 @@ public function register()
         /*
          * Add translation support to theme settings
          */
-        ThemeData::extend(function ($model) {
+        ThemeData::extend(static function ($model) {
             $model->extendClassWith(TranslatableModel::class);
 
             if (!$model->propertyExists('translatable')) {
                 $model->addDynamicProperty('translatable', []);
             }
 
-            $model->bindEvent('model.afterFetch', function() use ($model) {
+            // For some reason translatable property may appear in model's attributes
+            // and this is undesired behavior. If it happens we need to unset the attribute to prevent
+            // 'Unexpected type of array when attempting to save attribute "translatable"' exception
+            $model->bindEvent('model.saveInternal', static function() use ($model) {
+                if (isset($model->attributes['translatable']) && is_array($model->attributes['translatable'])) {
+                    unset($model->attributes['translatable']);
+                }
+            });
```

Comment:
```suggestion
```

Reply:
This is not needed when adding the Purgeable Behavior

Reply:
Yep, thanks! It's interesting, because I tried this before I worked on my specific solution and this trait did not help. Maybe anything else was wrong at the moment 😄 Anyway, now it's better and seems to be working.